src/main/java/seedu/address/ui/UiManager.java

```diff
@@ -18,12 +18,12 @@
 public class UiManager implements Ui {
 
     public static final String ALERT_DIALOG_PANE_FIELD_ID = "alertDialogPane";
-
+    private static MainWindow mainWindow;
```

Comment:
Seems like quite a big change to make this static, wonder if you could explain the rationale?